lib/diagramJSui.js

```diff
@@ -0,0 +1,34 @@
+import * as hooks from 'preact/hooks';
+
+import { html } from 'htm/preact';
+
+import {
+  render
+} from 'preact';
+
+
+/**
+ * A module that provides preact utilities.
+ */
+export default function DiagramJSui() {}
+
+/**
+ * Provides an html wrapper for preact.
+ */
+DiagramJSui.prototype.html = html;
+
+/**
+ *
+ * @param {Object} component the component to be rendered
+ * @param {Object} container the contaier to render the component into
```

Comment:
Let's specify the types here.

Reply:
Note that typescript types don't rely on inheritance, i.e. your type does not need to `instanceof` another type to work.

Anyway, as this function is a wrapper around `htm.render`, the types can be based on it:
`declare function render(tree: VNode, parent: HTMLElement): void;`

Indeed it's not `Component` but `VNode`.

Reply:
I'll fix this and then we can merge it.